{{cookiecutter.repo_name}}/CODE_OF_CONDUCT.md

```diff
@@ -1,49 +1,99 @@
+quick, friendly but clear word on an online forum or in person can help resolve
+a misunderstanding and de-escalate things.
+
+By embracing the following principles, guidelines and actions to follow or
+avoid, you will help us make {{cookiecutter.project_name}} a welcoming and productive community.
+
+1. **Be friendly and patient**.
+
+2. **Be welcoming**. We strive to be a community that welcomes and supports
+   people of all backgrounds and identities. This includes, but is not limited
+   to, members of any race, ethnicity, culture, national origin, color,
+   immigration status, social and economic class, educational level, sex, sexual
+   orientation, gender identity and expression, age, physical appearance, family
+   status, political belief, technological or professional choices, academic
+   discipline, religion, mental ability, and physical ability.
+
+3. **Be considerate**. Your work will be used by other people, and you in turn
+   will depend on the work of others. Any decision you take will affect users
+   and colleagues, and you should take those consequences into account when
+   making decisions. Remember that we're a world-wide community. You may be
+   communicating with someone with a different primary language or cultural
+   background.
+
+4. **Be respectful**. Not all of us will agree all the time, but disagreement is
+   no excuse for poor behavior or poor manners. We might all experience some
+   frustration now and then, but we cannot allow that frustration to turn into a
+   personal attack. It’s important to remember that a community where people
+   feel uncomfortable or threatened is not a productive one.
+
+5. **Be careful in the words that you choose**. Be kind to others. Do not insult
+   or put down other community members. Harassment and other exclusionary
+   behavior are not acceptable. This includes, but is not limited to:
+   * threats or violent language directed against another person
+   * discriminatory jokes and language
+   * posting sexually explicit or violent material
+   * posting (or threatening to post) other people's personally identifying
+     information ("doxing")
+   * personal insults, especially those using racist or sexist terms
+   * unwelcome sexual attention
+   * advocating for, or encouraging, any of the above behavior
+   * repeated harassment of others. In general, if someone asks you to stop,
+     then stop
+
+6. **Moderate your expectations**. Many in our community volunteer their time.
+   They are probably not purposefully ignoring issues, refusing to engage in
+   discussion, avoiding features, etc. but often just unavailable.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and MDAnalysis is no exception. It is important
+   that we resolve disagreements and differing views constructively. Remember
+   that we’re different. The strength of MDAnalysis comes from its varied community
```

Comment:
Should we remove "MDAnalysis" from here? Maybe replace with the project placeholder?

Reply:
Oops, yes, I missed this sorry!

Reply:
Now looks like:

https://github.com/lilyminium/test_mda_pip_rtd/blob/main/CODE_OF_CONDUCT.md

> Disagreements, both social and technical, happen all the time and Test Project with PyPI and ReadTheDocs is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of Test Project with PyPI and ReadTheDocs comes from its varied community that includes people from a wide range of backgrounds.